los_keeper_ros2/src/los_server/los_server.cc

```diff
@@ -17,7 +17,7 @@ los_keeper::ConvertToPointCloud(const PointCloudMsg &point_cloud_msg) {
   return pcl::PointCloud<pcl::PointXYZ>();
 }
 
-InputMsg los_keeper::ConverToInputMsg(const int drone_input) {
+InputMsg los_keeper::ConvertToInputMsg(const int drone_input) {
```

Comment:
The type of argument of this function will be "StatePoly".
TrajectoryPlanner outputs the trajectory (px, py, pz), and 
since the trajectory is expressed in polynomial, we can extract Jerk with time derivatives.

Reply:
Extraction from statepoly is dealt from wrapper, not los keeper. I set it as int as you did not implement the pure jerk input (not ros) datatype in type manager.

Reply:
https://github.com/orgs/LARR-Planning/projects/2/views/1?pane=issue&itemId=29843948